src/NeustaPimcorePresentationBundle.php

```diff
@@ -35,9 +25,4 @@ public function getInstaller()
     {
         return new Installer();
     }
-
-    protected function getComposerPackageName(): string
-    {
-        return 'teamneusta/pimcore-presentation-bundle';
-    }
```

Comment:
This is still necessary until we set Pimcore `10.5` as minimum version (currently it's still `10.0`).

Reply:
Args. You said that

Reply:
I think we should just raise the minimum version. There's no point in supporting Pimcore < 10.5 anymore.